File: 07_pgm_thread/src/co/edu/campusucc/poo/MainThreadExtends.java
package co.edu.campusucc.poo;

import java.io.*;
import java.util.*;

import co.edu.campusucc.poo.domain.ThreadExtends;

public class MainThreadExtends extends Thread {

    public static void main(String[] args) {
        ThreadExtends thread = new ThreadExtends();

        // Invoking Thread using start() method
        thread.start();
    }
}

File: python/java_to_python/07-pgmThread/src/co/edu/campusucc/poo/MainThreadExtends.py
import random
from src.co.edu.campusucc.poo import ThreadExtends

def main():
    thread = ThreadExtends()
    thread.setTimes(random.randint(0, 50))
    # Invoking Thread using start() method
    thread.start()

if __name__ == "__main__":
    main()